libensemble/executors/new_balsam_executor.py

```diff
@@ -0,0 +1,505 @@
+"""
```

Comment:
Should this file be `balsam_2_executor.py` and rename the old one to balsam_1_executor.py`?

Reply:
I've done: the new one is ``balsam_executor.py`` and the old one is ``legacy_balsam_executor.py``